config/crd/bases/nfd.kubernetes.io_nodefeaturediscoveries.yaml

```diff
@@ -4,7 +4,6 @@ kind: CustomResourceDefinition
 metadata:
   annotations:
     controller-gen.kubebuilder.io/version: v0.8.0
-    api-approved.kubernetes.io: unapproved, experimental-only
```

Comment:
How does this annotation get removed? Can we get it back in? 🧐

Reply:
`make generate manifest` did this

Reply:
Hmm, ok, can we prevent that auto-generation from doing this? In a follow-up PR that is?